app/containers/TunesContainer/tests/index.test.js

```diff
@@ -0,0 +1,81 @@
+/**
+ *
+ * Tests for TunesContainer
+ *
+ *
+ */
+
+import React from 'react';
+import { renderProvider, timeout } from '@utils/testUtils';
+import { TunesContainerTest as TunesContainer } from '../index';
+import { fireEvent } from '@testing-library/react';
+import { translate } from '@app/components/IntlGlobalProvider';
+
+describe('<TunesContainer /> container tests', () => {
+  let submitSpy;
+  let artistSearchBarId;
+
+  beforeEach(() => {
+    submitSpy = jest.fn();
+    artistSearchBarId = 'artist-search-bar';
+  });
+  it('should render and match the snapshot', () => {
+    const { baseElement } = renderProvider(<TunesContainer />);
+    expect(baseElement).toMatchSnapshot();
+  });
+
+  it('should trigger dispatchGetArtistSongs when a change is made in search bar', async () => {
+    const { getByTestId } = renderProvider(<TunesContainer dispatchGetArtistSongs={submitSpy} />);
+
+    fireEvent.change(getByTestId(artistSearchBarId), {
+      target: {
+        value: 'A'
+      }
+    });
+
+    await timeout(500);
```

Comment:
Will this test fail if we remove this timeout ?

Reply:
<img width="330" alt="Screenshot 2022-02-02 at 18 59 08" src="https://user-images.githubusercontent.com/98734275/152163079-9ca67ade-c9dd-4ac9-8aee-4340401a7a29.png">

Reply:
the submitSpy won't get called